backend/src/main/java/atlantbh/restaurants/models/filters/CategoryFilterBuilder.java

```diff
@@ -0,0 +1,11 @@
+package atlantbh.restaurants.models.filters;
+
+import atlantbh.restaurants.models.sortkeys.CategorySortKeys;
+import org.hibernate.Criteria;
+
+public class CategoryFilterBuilder extends BaseFilterBuilder<CategorySortKeys, CategoryFilterBuilder> {
+    @Override
+    protected Criteria addConditions(Criteria rootCriteria, boolean isCountCriteria) {
+        return null;
```

Comment:
return rootCriteria in this case

Reply:
Fixed in 8c841a20e489a31c66d177c8682339f9d4f19de0